### PR TITLE
feat(sworkstyle): add some more useful icons and replace some strange ones

### PR DESCRIPTION
**Pull request by boredland**
... remove a lot of spaces...

### CODE REVIEW

**Comment by Mathew-D** [comment]
You might want to add 'pcmanfm' = ''

File: community/sway/etc/skel/.config/sworkstyle/config.toml [hunk 1]
```diff
@@ -3,45 +3,51 @@
...
+'Alacritty' = ''
+'kitty' = ''
+'foot' = ''
+'font-manager' = ''
+'/Telegram/' = ''
+'/Stack Overflow/' = ''
+'/Bluetooth/' = ''
+'/Manjaro/' = ''
+'pamac-manager' = 'p'
+'/IntelliJ/' = ''
```
Comment (boredland) [inline community/sway/etc/skel/.config/sworkstyle/config.toml]:
```suggestion
'/IntelliJ/' = ''
'pcmanfm' = ''
```

**Comment by FallenChromium** [comment]
When I was introducing sworkstyle I was thinking if adding anything over defaults is reasonable, being that all people use different apps and I doubt we can ever cover them at least partially. That said, if we will anyway decide to add icons, I'm also adding some for myself :D. The problem is, sworkstyle has to scan the list every time Sway window tree gets updated, so it's a performance implication if it will get bigger. Probably adding a note about it is wise? My friend with a low-end laptop decided to disable sworkstyle fully because of that concern.

TL;DR My take is: it's ok to push it, but I'd prefer informing users on how to set it up to match their usecase rather than doing it for them.

**Comment by boredland** [comment]
As I get it, it scans the list top to bottom and uses the first match. So if we keep the defaults at a limited size in the top, the performance impact will be minimal. I will try and re-sort the list according to that.

**Comment by boredland** [comment]
And may I ask what device you friend is using? I have an original ThinkPad x200 I am using as a baseline - I'll give it a try soon to see if there's a performance impact.

**Comment by FallenChromium** [comment]
My friend uses a bit more fresh but pretty much low-end laptop. Hard drive, 2 cores of some random Pentium, 4GB of RAM, not absolutely terrible, but not great either.

Sometimes a bug happens with sworkstyle and it stalls one of the CPU cores to 100%, that happens when you kill sway and don't kill the sworkstyle, so two instances are running at the same time.

**Comment by boredland** [comment]
> Sometimes a bug happens with sworkstyle and it stalls one of the CPU cores to 100%, that happens when you kill sway and don't kill the sworkstyle, so two instances are running at the same time.

that is useful feedback and I think we had a similar issue with other tools. Will try and add a workaround for that.

**Comment by FallenChromium** [comment]
I have several problems with this PR, main being: terminal glyph is smaller compared to every other icon (probably because it's from font awesome and doesn't scale well), and nvim icon doesn't work now, though it's probably because of the default configuration. For terminal I'd recommend heading to [NerdFonts cheatsheet](https://www.nerdfonts.com/cheat-sheet) and using their hints to choose a bigger one, and for neovim, it uses capital `NVIM` in the title, so changing to that makes it work like a charm
Before:
image
After (I know the terminal icon is... A bit off. We can discuss that):
image

**Comment by FallenChromium** [comment]
The PCManFM icon also looks weird to me,
image

**Review by FallenChromium — APPROVED** [review]
I think it fits good with each other right now, I'd give it a "yes" 😃
